src/app/loading.ts

```diff
@@ -31,7 +31,7 @@ export let isFirstLoad = true;
 export function finishDAppLoading()  {
   const initialLoading = document.getElementById('evan-initial-loading');
 
-  if (initialLoading) {
+  if (initialLoading && initialLoading.className.indexOf('hidden') === -1) {
     utils.raiseProgress(10);
     initialLoading.className += ' hidden';
```

Comment:
you can use `initialLoading.add('hidden');` and avoid the check if it already has the class.

Reply:
I should use the classList add function within the statement. But the check should be included, so the inner statement will not be executed multiple times.